src/go/rpk/pkg/cli/cmd/cluster/partitions/status.go

```diff
@@ -0,0 +1,79 @@
+package partitions
+
+import (
+	"fmt"
+
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/api/admin"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func NewBalancerStatusCommand(fs afero.Fs) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "balancer-status",
+		Short: "Queries cluster for partition balancer status",
+		Long: `Queries cluster for partition balancer status:
+
+If continuous partition balancing is enabled, redpanda will continuously
+reassign partitions from unavailable nodes and from nodes using more disk space
+than configured limit.
+
+This command can be used to monitor the partition balancer status.
+
+FIELDS
+
+    Status:                        Either off, ready, starting, in_progress or stalled.
```

Comment:
So if these are the actual statuses, then write all in the same way--in this section, as either `<status_name>` or simply as plain text. So either:
`off`, `ready`, `starting`, `in_progress`, or `stalled`.
OR
off, ready, starting, in progress, or stalled.

Reply:
Going with off, ready, starting, in progress, or stalled.